### PR TITLE
Updates for Deploy V9.

### PR DESCRIPTION
**Pull request by AndyButland**
I've added what I believe are the necessary additions and changes for Deploy V9.

The main things added/changed are:

- A V9 page for configuration.
- V9 pages for the installation on new and existing sites.

I also reworked the "new site/existing site" pages, as I realised there's a fair bit repeated on them, so I've pulled that out into it's own page and linked to it from the "new" and "existing" articles.  With V9, we'd effectively repeat this 4 times, which I thought was going to be a bit of a pain to maintain, so having done this, whilst we still have 2 copies that are similar between versions, they do have some differences (and at least we don't have 4 copies!).

Other than reviewing the copy please take a careful look at the links I've added between pages, as I don't have a good way to test locally that these are correct.

### CODE REVIEW

File: Add-ons/Umbraco-Deploy/Installing-Deploy/Existing-site/index-v9.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+* Copy of views, css and scripts folder from production
+
+* CI/CD or Build Server that supports executing Powershell
+
+## Installation steps
+
+1. [Create copy of database and download views, CSS and scripts files from your production site](#install-umbraco-deploy-on-existing-site)
+2. [Set up Git repository and new Umbraco project](#set-up-git-repository-and-umbraco-project)
+3. [Install Umbraco Deploy via NuGet](#installing-and-setting-up-umbraco-deploy)
+4. [Configure CI/CD build server](#set-up-cicd-build-server)
```
Comment (jonat123) [inline Add-ons/Umbraco-Deploy/Installing-Deploy/Existing-site/index-v9.md:32]:
```suggestion
4. [Configure CI/CD build server](#set-up-cicd-pipeline)
```

File: Add-ons/Umbraco-Deploy/Installing-Deploy/Existing-site/index-v9.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+**/data/deploy*
+```
+
+### Installing and setting up Umbraco Deploy
+
+When Umbraco has been installed in a repository, we can go ahead to [install and configure Umbraco Deploy in the project](../Install-Configure/index-v9).
+
+### Set up CI/CD Pipeline
+
+At this stage your new website is prepared for use with Umbraco Deploy.  You should now move on to the setup of your [CI/CD build and deployment pipeline](../CICI-Pipeline).
```
Comment (jonat123) [inline Add-ons/Umbraco-Deploy/Installing-Deploy/Existing-site/index-v9.md:74]:
```suggestion
At this stage your new website is prepared for use with Umbraco Deploy.  You should now move on to the setup of your [CI/CD build and deployment pipeline](../CICD-Pipeline).
```

File: Add-ons/Umbraco-Deploy/Installing-Deploy/Existing-site/index-v9.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Once the build server has been set up you can start creating content and sync it between your environments. Make sure to follow the proper [deployment workflow](../../Deployments).
```
Comment (jonat123) [inline Add-ons/Umbraco-Deploy/Installing-Deploy/Existing-site/index-v9.md:76]:
```suggestion
Once the build server has been set up you can start creating content and sync it between your environments. Make sure to follow the proper [deployment workflow](../../Deployment-Workflow).
```

File: Add-ons/Umbraco-Deploy/Installing-Deploy/New-site/index-v9.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+### Installing and setting up Umbraco Deploy
+
+When Umbraco has been installed in a repository, we can go ahead to [install and configure Umbraco Deploy in the project](../Install-Configure/index-v9).
+
+### Set up CI/CD Pipeline
+
+At this stage your new website is prepared for use with Umbraco Deploy.  You should now move on to the setup of your [CI/CD build and deployment pipeline](../CICD-Pipeline).
+
+Once the build server has been set up you can start creating content and sync it between your environments. Make sure to follow the proper [deployment workflow](../../Deployments).
```
Comment (jonat123) [inline Add-ons/Umbraco-Deploy/Installing-Deploy/New-site/index-v9.md:75]:
```suggestion
Once the build server has been set up you can start creating content and sync it between your environments. Make sure to follow the proper [deployment workflow](../../Deployment-Workflow).
```

**Comment by jonat123** [comment]
@AndyButland It looks good, I have fixed a couple of links that did not seem to work correctly, there is also a conflict on this branch, if you can look at it and resolve it then i will merge this in 🙂

**Comment by AndyButland** [comment]
Thanks @jonat123 - changes applied and have merged latest from `main`, resolving conflicts.

**Comment by jonat123** [comment]
@AndyButland great! It seems that for some reason the vale linter is checking some other files and therefore says the checks have failed... I'll merge it in and see if it will cause any issues(I do not expect it to do so)

**Pull request merged**